src/containers/Review/ReviewSubmit.tsx

```diff
@@ -21,12 +25,29 @@ type ReviewSubmitProps = {
 
 const ReviewSubmit: React.FC<ReviewSubmitProps> = (props) => {
   const {
-    structure: { thisReview, assignment, canApplicantMakeChanges },
+    structure: { info, thisReview, assignment, canApplicantMakeChanges },
   } = props
+
+  const [fullyAssigned, setFullyAssigned] = useState(false)
+
+  const { data, loading, error } = useGetReviewableQuestionsQuery({
+    variables: {
+      applicationId: info.id,
+      stageId: thisReview?.current.stage.id as number,
+      levelNumber: assignment?.assigneeLevel as number,
+    },
+  })
```

Comment:
I thought we were going to include this info in the full structure so we don't have to query for it here?

Reply:
Yes, so I looked at this option but it would be tricky since we don't have the assignment (and which is the level) to pass on when the static part of the structure is queried, and I thought it was simpler to do this here

Reply:
What about in the "getReviewStructure" part? (You probably have a good reason, I'm just wondering)

Reply:
The logic on the getReviewStructure is way convoluted now to get this added to it... I just thought it would be simpler to do here. But it could be done there, I think